### PR TITLE
Feat: Add Export Record on Accouting Viewer

### LINKED ISSUE #1118: Exportar registros contables del documento.
### _**GIF**_

Peek 2023-06-20 14-39

### PR DESCRIPTION
**Pull request by elsiosanchez**
### _**GIF**_

Peek 2023-06-20 14-39

### _**ISSUES**_

Fixed: https://github.com/solop-develop/frontend-core/issues/1118

### CODE REVIEW

File: components/ADempiere/PanelInfo/Component/Accounting/index.vue [hunk 1]
```diff
@@ -499,17 +540,43 @@ export default defineComponent({
       }
     }
 
-    if (isEmptyValue(accoutingSchemaId.value)) {
+    function exportAccounting() {
+      const fileName = listAccouting.value.find(list => list.KeyColumn === accoutingSchemas.value).DisplayColumn
+      console.log({ fileName })
```
Comment (EdwinBetanc0urt) [inline components/ADempiere/PanelInfo/Component/Accounting/index.vue]: please remove this console

File: components/ADempiere/PanelInfo/Component/Accounting/index.vue [hunk 2]
```diff
@@ -499,17 +540,43 @@ export default defineComponent({
...
+      exportFileFromJson({
+        header: headerAccounting.value.map(a => a.label),
+        data: formatJson(headerAccounting.value.map(a => a.columnName), tableData.value),
+        fileName,
+        bookType: 'xlsx'
+      })
+    }
+
+    function formatJson(filterVal, jsonData) {
+      console.log({ filterVal, jsonData })
```
Comment (EdwinBetanc0urt) [inline components/ADempiere/PanelInfo/Component/Accounting/index.vue]: please remove this console

**Review by EdwinBetanc0urt — CHANGES_REQUESTED** [review]

File: components/ADempiere/PanelInfo/Component/Accounting/index.vue (hunk 2 above)
Comment (elsiosanchez) [reply components/ADempiere/PanelInfo/Component/Accounting/index.vue]: Resolve in next Commit https://github.com/solop-develop/frontend-default-theme/pull/681/commits/8ea0f49be74a586ca79fd7687f096a278307574b

File: components/ADempiere/PanelInfo/Component/Accounting/index.vue (hunk 1 above)
Comment (elsiosanchez) [reply components/ADempiere/PanelInfo/Component/Accounting/index.vue]: Resolve in next Commit https://github.com/solop-develop/frontend-default-theme/pull/681/commits/8ea0f49be74a586ca79fd7687f096a278307574b

**Review by EdwinBetanc0urt — APPROVED** [review]

**Pull request merged**